src/Standards/BestIt/Sniffs/Formatting/EqualOperatorSniff.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BestIt\Sniffs\Formatting;
+
+use BestIt\Sniffs\AbstractSniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Class EqualOperatorSniff.
+ *
+ * @author Mika Bertels <mika.bertels@bestit-online.de>
+ * @package BestIt\Sniffs\Formatting
+ */
+class EqualOperatorSniff extends AbstractSniff
+{
+    /**
+     * Is this sniff allowed to fix?
+     *
+     * @var bool $isFixable
+     */
+    private $isFixable = false;
+
+    /**
+     * Code when an equal operator was found.
+     *
+     * @var string CODE_EQUAL_OPERATOR_FOUND
+     */
+    public const CODE_EQUAL_OPERATOR_FOUND = 'EqualOperatorFound';
+
+    /**
+     * Warning when an equal operator was found.
+     *
+     * @var string WARNING_EQUAL_OPERATOR_FOUND
+     */
+    private const WARNING_EQUAL_OPERATOR_FOUND = '
+        Please check if you could use the "Identical" operator (===).
+    ';
+
+    /**
+     * Find the T_IS_EQUAL token and add a warning.
+     *
+     * @param File $phpcsFile
+     * @param int $stackPtr
+     *
+     * @return void
+     */
+    private function findToken(): void
+    {
+        $token = $this->token;
```

Comment:
Warum speicherst Du die Variable in $token? Du kannst doch auch direkt $this->token verwenden.